### PR TITLE
Add fluids recipe

### PR DESCRIPTION
**Pull request by volpatto**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/fluids/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bd91938b42cb03519b3a98e2e26af9e548cb9060ddd959420b2e9c07d53cc7c4
+
+build:
+  number: 0
```
Comment (isuruf) [inline recipes/fluids/meta.yaml]: Can this be `noarch: python`?

File: recipes/fluids/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - numpy >=1.5.0
```
Comment (isuruf) [inline recipes/fluids/meta.yaml]: You can remove this here and keep it in `run`

File: recipes/fluids/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    - pip
+    - python
+    - scipy >=0.9.0
```
Comment (isuruf) [inline recipes/fluids/meta.yaml]: Same here

File: recipes/fluids/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  summary: Fluid dynamics component of Chemical Engineering Design Library (ChEDL)
+  description: |
+    Fluids is open-source software for engineers and technicians working in the fields of chemical, mechanical, or civil engineering. It includes modules for piping, fittings, pumps, tanks, 
+    compressible flow, open-channel flow, and more.
+
+    The fluids library depends on the SciPy library to provide numerical constants, interpolation, integration, and numerical solving functionality. fluids runs on all operating systems which 
+    support Python, is quick to install, and is free of charge. fluids is designed to be easy to use while still providing powerful functionality. If you need to perform some fluid dynamics 
+    calculations, give fluids a try.
+  doc_url: https://fluids.readthedocs.io/
+  dev_url:
```
Comment (isuruf) [inline recipes/fluids/meta.yaml]: Can you add the github page url here?

File: recipes/fluids/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    - fluids
+
+about:
+  home: https://github.com/CalebBell/fluids
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: Fluid dynamics component of Chemical Engineering Design Library (ChEDL)
+  description: |
+    Fluids is open-source software for engineers and technicians working in the fields of chemical, mechanical, or civil engineering. It includes modules for piping, fittings, pumps, tanks,
```
Comment (isuruf) [inline recipes/fluids/meta.yaml]: Can you wrap text?

File: recipes/fluids/meta.yaml (hunk 1 above)
Comment (volpatto) [reply recipes/fluids/meta.yaml]: Sure, pure Python package, so it should works. Thanks!

File: recipes/fluids/meta.yaml (hunk 2 above)
Comment (volpatto) [reply recipes/fluids/meta.yaml]: Sure, thanks

File: recipes/fluids/meta.yaml (hunk 5 above)
Comment (volpatto) [reply recipes/fluids/meta.yaml]: Sorry, @isuruf. This one I didn't understand (sorry, this is my first experience with conda recipes). There is one example I could based myself to perform this wrap?

**Comment by volpatto** [comment]
@isuruf thank you for the review so far!

File: recipes/fluids/meta.yaml (hunk 5 above)
Comment (chrisburr) [reply recipes/fluids/meta.yaml]: The suggested is to insert new line characters to make them fit comfortably without sideways scrolling. Aiming for maximum line lengths of 80-100 characters is generally nice to have, i.e.:
```suggestion
    Fluids is open-source software for engineers and technicians working in
    the fields of chemical, mechanical, or civil engineering. It includes
    modules for piping, fittings, pumps, tanks, compressible flow, open-channel
    flow, and more.
```

Comment (volpatto) [reply recipes/fluids/meta.yaml]: Ah, nice! Thanks for the clarification, @chrisburr!

File: recipes/fluids/meta.yaml (hunk 4 above)
Comment (volpatto) [reply recipes/fluids/meta.yaml]: Sure, added.

File: recipes/fluids/meta.yaml (hunk 3 above)
Comment (volpatto) [reply recipes/fluids/meta.yaml]: Done, thanks

**Pull request merged**